api/user.json

```diff
@@ -1,3 +1,4 @@
 {
-  "name": "Johhny Tran"
+  "name": "Johhny Tran",
+  "is_admin?": false
```

Comment:
Why the question mark in the JSON key? I think the `is_` prefix is sufficient to know the user is an admin.